install-monorepo.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+
+# Print help and exit.
+function usage {
+	cat <<-EOH
+		usage: $0 
+
+		Installs all the required tooling for the Jetpack Monorepo. 
+	EOH
+	exit 1
+}
+
+if [[ $1 ]]; then
+	usage
+fi
+
+# Check if we're on a Mac or Linux, bail if we're not.
+OS="$(uname)"
+if [[ "$OS" == "Linux" ]]; then
+	ON_LINUX=1
+	sudo apt install build-essential
+elif [[ "$OS" != "Darwin" ]]; then
+	abort "Installer script is only supported on macOS and Linux."
+fi
+
+# Check of Homebrew and nvm are installed
+echo "Checking if Homebrew is installed..."
+if [[ -z "$(command -v brew)" ]]; then
+    echo "Installing Homebrew"
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+	if [[ ON_LINUX ]]; then # Add homebrew to PATH
+		echo 'eval "$(/home/ubuntu/.linuxbrew/bin/brew shellenv)"' >> /home/ubuntu/.profile
+		eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+		PATH=/home/linuxbrew/.linuxbrew/bin:$PATH
+		hash -r 
+	fi
+else
+	echo "Updating brew"
+    brew update
+	# Brew can be finicky on MacOS
+	if [ $? -ne 0 ]; then
```

Comment:
```suggestion
	if [[ $? -ne 0 ]]; then
```
The `[[` is a bash builtin, while `[` is an external program. The builtin is generally faster and less error-prone.